src/components/PostList/PostList.tsx

```diff
@@ -1,5 +1,17 @@
 import React from 'react';
+import { Post } from '../../types/Post';
+import { PostInfo } from '../PostInfo';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+type Props = {
+  posts: Post[];
+};
+
+export const PostList: React.FC<Props> = ({ posts }) => (
+  <>
+    <div className="PostList">
+      {posts.map(post => (
+        <PostInfo post={post} key={post.id} />
```

Comment:
Common practice is to set `key `as first prop
```suggestion
        <PostInfo key={post.id} post={post} />
```